book/website/reproducible-research/rdm/rdm-data-curation.md

```diff
@@ -7,34 +7,49 @@ Data curation promotes the use of data from the point of creation to ensuring th
 Data curation also enables you to easily access data sets and information that you need in an organised format, you can easily extract data from whichever repository it is stored. 
```

Comment:
Should the term 'data sets' or 'datasets' be used? I've seen both conventions - and while I don't have strong feelings about it, it it might be good to standardise. Also, maybe the phrase "where you can easily extra data [...]" could be used for clarity in the second half.

Reply:
Good point - I'll change that to datasets I think because it is easier to read/use it as one word